sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -372,12 +373,55 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
       assert(table.provider == Some("org.apache.spark.sql.hive.orc"))
     }
   }
+
+  private def checkOwner(db: String, owner: String,
```

Comment:
this doesn't check owner only, it checks the entire result of `DESCRIBE DATABASE EXTENDED`.

shall we simply do
```
val owner = sql(s"DESCRIBE DATABASE EXTENDED $db")
  .filter($"database_description_item" === "Owner Name")
  .collect().head.getString(1)
assert(owner == expected)
```